### PR TITLE
Rewrite Getting Started sections around Counter example and featuring most tools

### PR DESCRIPTION
**Pull request by chasefleming**
https://github.com/onflow/docs/issues/970

### CODE REVIEW

**Review by gregsantos — APPROVED** [review]

File: docs/build/getting-started/fcl-quickstart.md [hunk 1]
```diff
@@ -5,157 +5,502 @@ sidebar_label: Simple Frontend
...
-```
+```bash
 npm install @onflow/fcl --save
 ```
 
-Next we'll want to add to our FCL configuration. There's a lot you can do here, but for this simple example, let's configure `accessNode.api` to talk to the Testnet Flow Access Node. An Access Node serves as the primary point of interaction for clients, such as wallets, dapps, and other services, to communicate with the Flow network. It provides a gateway for these clients to submit transactions, query data, and retrieve information without having to connect to the entire network or maintain a full copy of the blockchain.
+This command installs FCL and adds it to your project's dependencies.
 
-For our example, we are going to point at a free Access Node provided by Flow. Add the following config code to your `src/App.js`:
+Next, we'll configure FCL to connect to the Flow Testnet. An Access Node serves as the primary point of interaction for clients to communicate with the Flow network. It provides a gateway for submitting transactions, querying data, and retrieving information.
```
Comment (briandoyle81) [inline docs/build/getting-started/fcl-quickstart.md:71]: If we have links we can use for [Flow Testnet] and [Access Node], I'd add them here.

**Comment by chasefleming** [comment]
@briandoyle81 feedback has been addressed

File: docs/build/getting-started/fcl-quickstart.md [hunk 2]
```diff
@@ -5,157 +5,502 @@ sidebar_label: Simple Frontend
...
+
+Now, let's create a function to query the greeting from the blockchain:
+
+```jsx
+const queryGreeting = async () => {
+  const res = await fcl.query({
+    cadence: `
+      import HelloWorld from 0xa1296b1e2e90ca5b
+
+      pub fun main(): String {
```
Comment (briandoyle81) [inline docs/build/getting-started/fcl-quickstart.md:136]: @chasefleming I think this is the old way of doing this.  Shouldn't it be access(all)?

I'm getting:

 HTTP Request Error: An error occurred when interacting with the Access API.
      error=Invalid Flow argument: failed to execute script: [Error Code: 1101] failed to execute script at block (7ab1a2d983669a1aa3e9bade1270de00316ad98477cae5093e57ca741b170562): [Error Code: 1101] error caused by: 1 error occurred:
	* [Error Code: 1101] cadence runtime error: Execution failed:
error: `pub` is no longer a valid access keyword
 --> de8a43fd478ffdd20eae82630857d7bdd680c13673bfa37bc41a859a32d654a8:4:8
  |
4 |         pub fun main(): String {
  |         ^^^

File: docs/build/getting-started/fcl-quickstart.md [hunk 3]
```diff
@@ -5,157 +5,502 @@ sidebar_label: Simple Frontend
...
+  const [greeting, setGreeting] = useState('');
+  const [user, setUser] = useState({ loggedIn: false });
+  const [newGreeting, setNewGreeting] = useState('');
+
+  const queryGreeting = async () => {
+    const res = await fcl.query({
+      cadence: `
+        import HelloWorld from 0xa1296b1e2e90ca5b
+
+        pub fun main(): String {
```
Comment (briandoyle81) [inline docs/build/getting-started/fcl-quickstart.md:404]: access(all)

**Review by briandoyle81 — DISMISSED** [review]
Improvements are great but ran into a few errors.  Left comments!

**Pull request merged**